main/main.js

```diff
@@ -44,7 +45,10 @@ function createWindow () {
       slashes: true
     }))
   })
-
+  // If we're in development mode then create an Electron Connect client for live reload.
+  if (!process.defaultApp) {
```

Comment:
the defaultApp runs when in dev mode, so the condition should be:
```suggestion
  if (process.defaultApp) {
```